cmd/util/cmd/execution-state-extract/cmd.go

```diff
@@ -108,42 +108,6 @@ func run(*cobra.Command, []string) {
 		if err != nil {
 			log.Fatal().Err(err).Msg("invalid state commitment length")
 		}
-	} else if !flagNoMigration {
```

Comment:
[these changes are no longer exists on master](https://github.com/onflow/flow-go/pull/1813/files#diff-eaa1a5716677c1fe0529fa5df3c6d878cf0b0452e5c2b1dbb2a4f103190f83a8R111).

could you confirm @m4ksio